Sources/PrinceOfVersions/PrinceOfVersions.swift

```diff
@@ -231,7 +231,10 @@ private extension PrinceOfVersions {
             let decoder = JSONDecoder()
             decoder.keyDecodingStrategy = .convertFromSnakeCase
 
-            let updateInfo = try decoder.decode(UpdateInfo.self, from: data)
+            var updateInfo = try decoder.decode(UpdateInfo.self, from: data)
+            /// due to: https://github.com/infinum/iOS-prince-of-versions/issues/38
+            /// `PovError.validate(:)` falis if proper configuration is not selected, and it can't be selected without proper `userRequirements`
+            updateInfo.userRequirements = options.userRequirements
```

Comment:
```suggestion
            updateInfo.userRequirements = options.userRequirements
```

As this part of the code look ok and logical, I would suggest removing this comment. However, If you want to keep it, then just replace documentation comment `///` with regular codebase comment `//` :)